### ISSUE TITLE
MSYS2 / Git broken in Windows 11 Canary build 27744

### ISSUE THREAD

**Issue by snickler** [msys2/msys2.github.io#353]
Currently experiencing this on the Arm64 version of Windows, but it may also apply to x64 versions of Windows.

Both the standalone MSYS2 and the one bundled with Git for Windows fails to load in Windows 11 Canary build 27744. Posting this issue to bring awareness to anyone how may happen to be on the Windows Insider Canary builds and have noticed anything related to using MSYS2 failing to load. 

https://x.com/sinclairinat0r/status/1855733791165169916

**Comment by lazka** [comment]
@dscho fyi

**Comment by dscho** [comment]
> edit: seems to be "fixed" though

Yep, I fast-tracked this in https://github.com/git-for-windows/msys2-runtime/pull/76, and did not have time to backport it to MSYS2, too. Will do so right now.

**Comment by lazka** [comment]
Ah, no prob, I can do too if needed.
